enterprise/backend/src/metabase_enterprise/metabot_v3/tools/find_outliers.clj

```diff
@@ -1,8 +1,43 @@
 (ns metabase-enterprise.metabot-v3.tools.find-outliers
   (:require
+   [medley.core :as m]
+   [metabase-enterprise.metabot-v3.client :as metabot-v3.client]
    [metabase-enterprise.metabot-v3.tools.interface :as metabot-v3.tools.interface]
-   [metabase.util.malli :as mu]))
+   [metabase.api.common :as api]
+   [metabase.lib.types.isa :as lib.types.isa]
+   [metabase.query-processor :as qp]
+   [metabase.util :as u]
+   [metabase.util.malli :as mu]
+   [toucan2.core :as t2]))
 
 (mu/defmethod metabot-v3.tools.interface/*invoke-tool* :metabot.tool/find-outliers
-  [_tool-name _arguments _env]
-  {:output "Not implemented."})
+  [_tool-name {{:keys [query_id
+                       report_id
+                       metric_id
+                       result_field_id]}
+               :data-source} _env]
+  (when (or query_id report_id result_field_id)
+    (throw (ex-info "Not implemented" {})))
+  (let [{:keys [dataset_query]} (api/read-check (t2/select-one [:model/Card :collection_id :dataset_query] metric_id))
+        {:keys [data status]} (qp/process-query dataset_query)]
```

Comment:
What about running this as a user land query? Then it would be included in the statistics, participate in cacheing, etc.